bin/cleaver

```diff
@@ -24,6 +29,7 @@ function createAndSave(files, options) {
         .then(function (product) {
           var outputFile = presentation.options.output || path.basename(file, '.md') + '-cleaver.html';
           fs.writeFile(outputFile, product);
+          done();
```

Comment:
nit: we can change this to `done && done()` instead of defining it as a no-op above